bin/run_tests.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 ## Call each task's testing suite
 ## TODO: Add more logging output and possibly make asynchronus
-for task in $(ls -d tasks/* | egrep -v "dr_dbutils|pg_utils|shared_libraries|package")
+for task in $(ls -d tasks/* | egrep -v "shared_libraries|package")
```

Comment:
`shared_libraries` should also be removed as it no longer resides in the `tasks` directory nor is it built there like the `packages` directory.

Reply:
Removed